kiwi/package_manager/dnf.py

```diff
@@ -184,8 +184,11 @@ def process_delete_requests(self, force=False):
         else:
             chroot_dnf_args = self.root_bind.move_to_root(self.dnf_args)
             return Command.call(
-                ['chroot', self.root_dir, 'dnf'] + chroot_dnf_args +
-                self.custom_args + ['autoremove'] + delete_items,
+                [
+                    'chroot', self.root_dir, 'dnf'
+                ] + chroot_dnf_args + self.custom_args + [
+                    'autoremove'
+                ] + delete_items,
```

Comment:
what kind of complain would fix this?

Reply:
line break before/after operator

Reply:
I am missing something, as I have the last version from git of each package for flake:

* flake8-3.6.0
* pyflakes-2.0.0
* pycodestyle-2.4.0
* mccabe-0.6.1

and no complains for this:

```
def xx():
    lx = (1 +
          2)
    print(lx)

    xx(['a'] +
       ['b'])
```

You can test it also using the last released version

```
python3 -m venv
source venv/bin/activate
pip install flake8
flake8 ./test.py
```